charts/camunda-platform/values.yaml

```diff
@@ -1941,7 +1942,7 @@ elasticsearch:
   resources:
     requests:
       cpu: 1
-      memory: 1Gi
```

Comment:
as the heap is already 1G we need to request 2G, see ["Set Xms and Xmx to no more than 50% of your total memory."](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/advanced-configuration.html#set-jvm-heap-size)